src/digest.rs

```diff
@@ -448,8 +447,66 @@ pub extern fn SHA512_4(out: *mut u8, out_len: c::size_t,
     polyfill::slice::fill_from_slice(out, digest);
 }
 
+unsafe extern fn sha1_block_data_order(state: *mut u64, data: *const u8, num: c::size_t) {
```

Comment:
Sorry this isn't in the documentation: Please limit line length to 80 characters (punctuation can spill over by a character or two).